CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ TRIBITS_ADD_ETI_INSTANTIATIONS(TpetraCore
 
```

Comment:
We need to add a test that ensures no trailing whitespace was introduced in a new commit.